src/us/kbase/workspace/kbase/WorkspaceServerMethods.java

```diff
@@ -96,45 +92,24 @@
 	
 	final private Workspace ws;
 	final private Types types;
-	final private URL handleServiceUrl;
 	final private ConfigurableAuthService auth;
 	private final IdReferenceHandlerSetFactoryBuilder idFacBuilder;
 	
 	public WorkspaceServerMethods(
 			final Workspace ws,
 			final Types types,
 			final IdReferenceHandlerSetFactoryBuilder idFacBuilder,
-			final URL handleServiceUrl,
 			final ConfigurableAuthService auth) {
 		this.ws = ws;
 		this.types = types;
 		this.idFacBuilder = idFacBuilder;
-		this.handleServiceUrl = handleServiceUrl;
 		this.auth = auth;
 	}
 	
 	public ConfigurableAuthService getAuth() {
 		return auth;
 	}
 	
-	public URL getHandleServiceURL() {
```

Comment:
Why was this removed?  Where is that call now?

Reply:
Maybe I get it... There is still a method in the config part.

Reply:
the handle service stuff living in that class was cruft from older implementations - there was no real reason for it to be there, and it made more sense just to move all that code to the main server class.

Reply:
And yeah, the main server class is now getting a client from the object returned by `initWorkspaceService` or whatever it's called, rather than pulling the url from this class.